modules/addbook.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
+export const addBook = () => {
+  const contact = document.getElementById('contacts');
+  const bookList = document.getElementById('booklist');
+  const addBook2 = document.getElementById('addBook');
+  contact.style.display = 'none';
+  bookList.style.display = 'none';
+  addBook2.style.display = 'block';
+};
```

Comment:
- [ ] This linter rule can easily be followed, hence I would recommend you enable it. Linters help us write standardized code, let's maintain that. I would give you an example to enable this rule and fix the issue.

  In this file:
  ```
  const addBook = () => {
    const contact = document.getElementById('contacts');
    const bookList = document.getElementById('booklist');
    const addBook2 = document.getElementById('addBook');
    contact.style.display = 'none';
    bookList.style.display = 'none';
    addBook2.style.display = 'block';
  };
  
  export default addBook;
  ```
  Then in the `main.js` file where you import `addBook` function, simply import it as default by changing this line:
  ```
  import { addBook } from './addbook.js';
  ```
  to this line:
  ```
  import addBook from './addbook.js';
  ```

Reply:
Thank you, @TedLivist for helping and all corrections in my project. I have removed all the linter disabling as you mentioned.